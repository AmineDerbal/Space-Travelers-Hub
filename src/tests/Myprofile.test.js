import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Myprofile from '../Components/Myprofile';

it('renders My Profile correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <Myprofile />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
