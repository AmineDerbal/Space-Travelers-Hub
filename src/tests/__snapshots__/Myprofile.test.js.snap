// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders My Profile correctly 1`] = `
<div
  className="profile-content"
>
  <div
    className="missions-list"
  >
    <h2>
      My Missions
    </h2>
    <table
      className="table table-striped table-bordered"
    >
      <tbody />
    </table>
  </div>
  <div
    className="rockets-list"
  >
    <h2>
      My Rockets
    </h2>
    <table
      className="table table-striped table-bordered"
    >
      <tbody />
    </table>
  </div>
</div>
`;

exports[`renders My correctly 1`] = `
<div
  className="profile-content"
>
  <div
    className="missions-list"
  >
    <h2>
      My Missions
    </h2>
    <table
      className="table table-striped table-bordered"
    >
      <tbody />
    </table>
  </div>
  <div
    className="rockets-list"
  >
    <h2>
      My Rockets
    </h2>
    <table
      className="table table-striped table-bordered"
    >
      <tbody />
    </table>
  </div>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="profile-content"
>
  <div
    className="missions-list"
  >
    <h2>
      My Missions
    </h2>
    <table
      className="table table-striped table-bordered"
    >
      <tbody />
    </table>
  </div>
  <div
    className="rockets-list"
  >
    <h2>
      My Rockets
    </h2>
    <table
      className="table table-striped table-bordered"
    >
      <tbody />
    </table>
  </div>
</div>
`;
